### PR TITLE
CLOUDP-221643: Document the meaning of read/write only fields

### PR DESCRIPTION
**Pull request by lovisaberggren**
## Description

Added documentation on the `read only` and `write only` fields as a follow-up from a previous PR.

Previous change: [CLOUDP-219156](https://jira.mongodb.org/browse/CLOUDP-219156)

### CODE REVIEW

**Review by matt-condon — APPROVED** [review]

File: docs/doc_1_concepts.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ Note: The `experimental` label refers specifically to the Atlas Go SDK methods.
 
 If you encounter any problems with methods marked as experimental, feel free to raise a [Github issue](https://github.com/mongodb/atlas-sdk-go/issues/new/choose).
 
+## Read Only and Write Only Fields
+
+Some fields are documented as `read only` or `write only` in the [documentation](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_last_reference.md#documentation-for-models) and the Go doc comments.
```
Comment (wtrocki) [inline docs/doc_1_concepts.md:60]:
```suggestion
Each SDK Request and Response might contain read or write only fields. 
Those fields are identified by godoc comments and [documentation](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_last_reference.md#documentation-for-models).
```

File: docs/doc_1_concepts.md [hunk 2]
```diff
@@ -55,6 +55,13 @@ Note: The `experimental` label refers specifically to the Atlas Go SDK methods.
...
+
+- `read only` fields are included in responses but not in requests
```
Comment (wtrocki) [inline docs/doc_1_concepts.md:62]:
```suggestion
We use godoc comments to annotate fields as read and write only:
- `// Read only field.` comment means that field can be included in responses but not in requests
```

File: docs/doc_1_concepts.md [hunk 3]
```diff
@@ -55,6 +55,13 @@ Note: The `experimental` label refers specifically to the Atlas Go SDK methods.
...
+- `write only` fields may be included in request but not in responses
```
Comment (wtrocki) [inline docs/doc_1_concepts.md:63]:
```suggestion
- `// Write only field.` means that field may be included in requests but not in responses
```

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Some minor and optional suggestions for your consideration

File: docs/doc_1_concepts.md (hunk 3 above)
Comment (wtrocki) [reply docs/doc_1_concepts.md:63]: Write only field is an marker on it's own that could be used by tooling or linter in the future.

**Comment by lovisaberggren** [comment]
> Some minor and optional suggestions for your consideration

Thanks for the suggestions, made some minor changes.

File: docs/doc_1_concepts.md [hunk 4]
```diff
@@ -55,6 +55,15 @@ Note: The `experimental` label refers specifically to the Atlas Go SDK methods.
 
 If you encounter any problems with methods marked as experimental, feel free to raise a [Github issue](https://github.com/mongodb/atlas-sdk-go/issues/new/choose).
 
+## Read Only and Write Only Fields
+
+Each SDK request and response might contain read or write only fields identified by the [documentation](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_last_reference.md#documentation-for-models) and the Go doc comments.
```
[the remark this answers is not in the document]
Comment (lovisaberggren) [reply docs/doc_1_concepts.md]: Cool thanks, added!

**Comment by wtrocki** [comment]
Catching up with PR. All LGTM.

**Pull request merged**